WhiteDentalClinic/WhiteDentalClinic.Api/Controllers/AppointmentsController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WhiteDentalClinic.Application.Models;
+using WhiteDentalClinic.Application.Models.AppointmentModel;
+using WhiteDentalClinic.Application.Services.Interfaces;
+
+namespace WhiteDentalClinic.Api.Controllers
+{
+    public class AppointmentsController : ApiController
+    {
+        readonly IAppointmentService _appointmentService;
+        public AppointmentsController(IAppointmentService appointmentService)
+        {
+            _appointmentService = appointmentService;
+        }
+
+        [HttpGet("customers{customerId}")]
```

Comment:
These endpoints are currently looking like this:
![image](https://user-images.githubusercontent.com/117767491/230028252-3d4297c1-d304-4809-89fc-7bfe902ccea8.png)
There are some issues here:
1. no naming convention is used (small capital letter mixed with upper capital letter - choose one and stick to it)
2. missing slash before {customerId} and {dentistId} (is should be /api/appointments/customers/{customerId} and so on). I'm kind of surprised it works this way 😄 
3. there is no get all appointments endpoint available (not exactly an issue if it not necessary but wanted to point it out)